packages/css/src/__tests__/generate-compression-map.test.ts

```diff
@@ -0,0 +1,139 @@
+    ._1n2onqa1>a:active, ._1k4fnqa1>a[aria-current=page]{-webkit-text-decoration-style:solid;text-decoration-style:solid}
+    ._1nrm18uv:active, ._1a3b18uv:focus, ._9oik18uv:hover{-webkit-text-decoration-color:initial;text-decoration-color:initial}
+    ._1nrm18uv:active, ._9oik18uv:hover, ._5bd618uv:link, ._1ydc18uv:visited{-webkit-text-decoration-color:initial;text-decoration-color:initial}
+    ._1ohyglyw:active, ._49pcglyw:focus, ._ra3xglyw:focus-visible, ._ksodglyw:hover{outline-style:none}
+    ._1ohyglyw:active, ._ksodglyw:hover, ._q4asglyw:link, ._tpgfglyw:visited{outline-style:none}
+    ._1oxgru3m:active{transition-duration:0s}
+    ._9h8h1e9r:active, ._f8pj1e9r:focus, ._30l31e9r:hover, ._10531e9r:visited{color:var(--ds-text-subtlest,#7a869a)}
+    @media screen and (min-width:1300px){._1jhpoyl8{max-width:10vw}}
+    @media (max-width:1199px){._11usglyw{display:none}}
+    @media (min--moz-device-pixel-ratio:2){._11y7oza4{max-width:510px}._11y7uu9g{max-width:840px}._l82t7vkz{border-left-width:1pc}._j7o07vkz{border-right-width:1pc}._1od57vkz._1od57vkz{border-left-width:1pc}._l82tgktf{border-left-width:20px}._j7o0gktf{border-right-width:20px}._yksp1ssb{width:50%}._1b421ssb{height:50%}._s8ks18ws{transform:scale(2)}._u1wz1nty{transform-origin:0 0}}
+    @media (min-width:1000px) and (max-width:1439px){._hnu8tcjq{display:block!important}}
+    @media (min-width:1200px){._jvpg11p5{display:grid}._1nwdwxkt{grid-template-columns:1fr 1fr}._1vlxckbl{grid-gap:3pc}._kz8c16xz{padding-top:6pc}._1jyu16xz{padding-right:6pc}._11et16xz{padding-bottom:6pc}._fgkv16xz{padding-left:6pc}._szna1wug{margin-top:auto}._13on1wug{margin-right:auto}._1f3k1wug{margin-bottom:auto}._inid1wug{margin-left:auto}._12wp9ac1{max-width:1400px}._jvpgglyw{display:none}}
+    @media (min-width:1440px) and (max-width:1919px){._pbi4tcjq{display:block!important}}
+    @media (min-width:1920px) and (max-width:2559px){._16b9tcjq{display:block!important}}
+    @media (min-width:2560px) and (max-width:2999px){._jmaqtcjq{display:block!important}}
+    @media (min-width:3000px){._1q5htcjq{display:block!important}}
+    @media (min-width:800px) and (max-width:999px){._11x1tcjq{display:block!important}}
+    @media (min-width:800px){._121jagmp{display:none!important}}
+    @media (prefers-reduced-motion:reduce){._1bumglyw{animation:none}._sedtglyw{transition:none}}
+    @media screen and (-webkit-min-device-pixel-ratio:0){._14kw1hna >textarea{word-break:break-word}._mc2h1hna{word-break:break-word}}
+    @media screen and (-webkit-transition){._14fy1hna{word-break:break-word}._1vdp1hna >textarea{word-break:break-word}}
+    @media screen and (max-height:400px){._17gjpfqs{position:static}}
+    @media screen and (min-width:1300px){._1jhpoyl8{max-width:10vw}}
+`;
+
+  const baseResult = {
+    '154i14e6': 'a',
+    '14tk72c6': 'b',
+    '14n4stnw': 'c',
+    '13h81y44': 'd',
+    '1di6k6hx': 'e',
+    irr3k6hx: 'f',
```

Comment:
Would it make a difference if we change all of these keys (irr3k6hx, jomrk6hx, etc) to be in single quotes? I'm not quite sure why some are in single quotes and some are not

Reply:
Ohh I think I see why, the number(s) at the start of the key means that the single quotes are required right?

Reply:
Would personally prefer consistency, i.e. all of the keys being in single quotes

Reply:
The formatting is automatically done by [prettier](https://prettier.io/blog/2019/04/12/1.17.0.html#add-an-option-to-modify-when-prettier-quotes-object-properties-5934-by-azz). We don't customize this rule so it's the default "as-needed". I am not particular about the formatting but happy to change it if you prefer.

Reply:
Ah no worries, thank you for clarifying! Since it's being done by our linter, I'd be happy to leave it as-is